setup.py

```diff
@@ -10,12 +10,11 @@
 
 install_requires = [
     'sqlparse >= 0.3.0',
-    'google-cloud >= 0.34.0',
```

Comment:
@odeke-em, @busunkim96 asked me to remove this dependency. Hope that's ok with you.

Reply:
For context, this is an old package (last published July 2018) https://pypi.org/project/google-cloud/. The cloud libraries used to all be published under this single package but were split out.

Reply:
Gotcha, thanks! In that case, can we replace it with the appropriate cloud-Spanner import?

Reply:
Oh never mind, we already Gabe the google-cloud-spanner dependency. Resolving this conversation as I’ve answered my own question.